packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/actions/assistant/message.operation.ts

```diff
@@ -63,6 +60,46 @@ const properties: INodeProperties[] = [
 			},
 		},
 	},
+	{
+		displayName: 'Memory',
+		name: 'memory',
+		type: 'options',
+		options: [
+			{
+				// eslint-disable-next-line n8n-nodes-base/node-param-display-name-miscased
```

Comment:
Why disable this rule?

Reply:
To be consistent with the `prompt` parameter. Otherwise each name would have to be in Capital Case.